spec/models/manageiq/providers/google/cloud_manager/refresher_spec.rb

```diff
@@ -0,0 +1,257 @@
+require "spec_helper"
+require 'fog/google'
+
+describe ManageIQ::Providers::Google::CloudManager do
```

Comment:
As we noticed in other refresher_spec file... This described class here should be `::Refresher`